src/chain.h

```diff
@@ -18,6 +18,7 @@
 static const int SPROUT_VALUE_VERSION = 1001400;
 static const int SAPLING_VALUE_VERSION = 1010100;
 static const int CHAIN_HISTORY_ROOT_VERSION = 2010200;
+static const int NU5_DATA_VERSION = 4050000;
```

Comment:
The current release's version is `4040100`; the new data will start being persisted to disk from v4.5.0-rc1 onward.

Reply:
Suggestion: ZIP244_DATA_VERSION

Reply:
It's going to enable storing more data from NU5 (specifically `nOrchardValue` which would be ZIP 224).